metaflow/plugins/pypi/fast_bootstrap.py

```diff
@@ -88,6 +58,9 @@ def install_fast_initializer(architecture):
             return fast_initializer_path
 
         # TODO: take architecture into account
+        url = CONDA_FAST_INIT_BIN_URL
```

Comment:
With this change, does the TODO above still make sense? Or are we going to handle the architecture at a different level, by setting `CONDA_FAST_INIT_BIN_URL` to an architecture-specific url?

Reply:
Fair point. The client needs to have knowledge of the target architecture either way as it is the one doing the solving for packages. The client also provides the architecture input for the bootstrap script, so we might as well keep the URL defining logic completely client-side.